chef/cookbooks/monasca/attributes/default.rb

```diff
@@ -36,6 +40,12 @@
 
 default[:monasca][:api][:bind_host] = "*"
 
+# metric-agent default service settings
+default[:monasca][:metric_agent][:service_name] = monasca_metric_agent_service_name
+default[:monasca][:metric_agent][:user] = "monasca-agent"
+default[:monasca][:metric_agent][:group] = "monasca-agent"
```

Comment:
This should be monasca. As we discussed with other guys from community about rpm packaging.